### PR TITLE
fix(toaster): fix toaster aspect and position

### PR DESCRIPTION
**Pull request by ricardogoncalves89**

### CODE REVIEW

File: packages/ui-components/src/components/toaster/toaster.scss [hunk 1]
```diff
@@ -21,10 +22,11 @@
...
 	 */
 
-	--toaster-large-height: 74px;
-	--toaster-small-height: 58px;
+	--toaster-large-height: 72px;
+	--toaster-small-height: 56px;
 
-	--toaster-width: 624px;
+	--toaster-width: 500px;
+	--toaster-top-space: 16px;
```
Comment (pedrorebelocosta) [inline packages/ui-components/src/components/toaster/toaster.scss:29]:
```suggestion
	--toaster-top-space: #{$spacing-4x};
```

Comment (miguelpinto98) [inline packages/ui-components/src/components/toaster/toaster.scss:29]: should this action propagate over all application?

**Review by pedrorebelocosta — CHANGES_REQUESTED** [review]
just a small suggestion

File: packages/ui-components/src/components/toaster/toaster.scss (hunk 1 above)
Comment (ricardogoncalves89) [reply packages/ui-components/src/components/toaster/toaster.scss:29]: yes

**Review by miguelpinto98 — APPROVED** [review]

**Review by pedrorebelocosta — APPROVED** [review]

**Review by GoncaloCamaz — APPROVED** [review]

**Pull request merged**